pkg/recommender/types.go

```diff
@@ -15,7 +15,7 @@
 package recommender
 
 import (
-	"github.com/goph/logur"
+	"github.com/banzaicloud/cloudinfo/pkg/cloudinfo-client/models"
```

Comment:
The generated client should not be referenced by the application. Rather use the types (structs) that are defined in the application.

Eg.: models.ProductDetails -> VirtualMachine